.travis.yml

```diff
@@ -1,12 +1,19 @@
+dist: xenial
+sudo: false
+
 language: python
 python:
   - "2.7"
-  - "3.4"
   - "3.5"
   - "3.6"
-  - "pypy-5.4.1"
```

Comment:
Any reason we remove 3.4 and pypy 5.4.1?

Reply:
Because of Travis' support (also 3.4 has been EOL'd a while ago)

Reply:
Python 3.4 is not available anymore with that xenial image? It is EOL since march, but the python driver still support it until we decide to remove it (next major probably).

Reply:
IIRC that's correct xenial doesn't have it.

Reply:
aha, and xenial doesn't have 3.7 :wink:

Reply:
Alright... so let's keep xenial and 3.7 then.